gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -286,6 +287,25 @@ get_mode (void)
     return GIS_DRIVER_MODE_NEW_USER;
 }
 
+static gboolean
+connect_to_gdm (void)
+{
+  GdmClient *client;
```

Comment:
```suggestion
  g_autoptr(GdmClient) client = NULL;
```

(if that compiles, else unref on the way out)